SPECS/nodejs/nodejs18.spec

```diff
@@ -1,229 +0,0 @@
-# Retrieved from 'deps/npm/package.json' inside the sources tarball.
-%define npm_version 9.8.1
-
-Summary:        A JavaScript runtime built on Chrome's V8 JavaScript engine.
-Name:           nodejs18
```

Comment:
Please also update the licenses and (potentially) `cgmanifest.json`.

Reply:
addressed comment